components/eventing-controller/controllers/subscription/beb/reconciler.go

```diff
@@ -869,29 +884,30 @@ func isInDeletion(subscription *eventingv1alpha1.Subscription) bool {
 const timeoutRetryActiveEmsStatus = time.Second * 30
```

Comment:
What do you think about moving all the consts up in the file in one block?

Reply:
moved